src/styles/page-template/index.md.njk

```diff
@@ -36,161 +36,138 @@ You can customise the page template, for example, by adding a service name and n
 
 {{ example({group: "styles", item: "page-template", example: "custom", customCode: true, html: true, nunjucks: true, open: false, size: "xl" }) }}
 
-## Nunjucks
+## Changing template content
 
+If you’re [using Nunjucks](https://frontend.design-system.service.gov.uk/use-nunjucks-macros/), you can change the template’s content using options.
 
-### Configuring Nunjucks
+You can use either:
 
-When using Nunjucks you’ll need to add `node_modules/govuk-frontend/`
-as available views:
+- `set` to use a single value or string
```

Comment:
[`set`](https://mozilla.github.io/nunjucks/templating.html#set) and [`block`](https://mozilla.github.io/nunjucks/templating.html#block) do two different things, and can't be used interchangeably.

`set` is used to set variables, and `block` is used to define content for blocks.

For example, in this [line from the template](https://github.com/alphagov/govuk-frontend/blob/02bebd779916d44ed8c623785cfcf44f947529e1/src/govuk/template.njk#L10):

```twig
<title{% if pageTitleLang %} lang="{{ pageTitleLang }}"{% endif %}>{% block pageTitle %}GOV.UK - The best place to find government services and information{% endblock %}</title>
```

`pageTitleLang` is a _variable_, which can only be changed using `set`. If you wanted to change it from your own template that extends `template.njk`, you'd do:

```twig
{% set pageTitleLang = "cy" %} ✅
```

Using `block` here **would not work**:

```twig
{% block pageTitleLang %}cy{% endblock %} ❌
```

`pageTitle` meanwhile is a block, and so you'd set it using the `block` syntax:

```twig
{% block pageTitle %}GOV.UK{% endblock %} ✅
```

Again, setting the pageTitle using `set` here **would not work**:
```twig
{% set pageTitle = "GOV.UK" %} ❌
```

Reply:
I think there is some logic around when we use blocks and when we use variables – variables are generally used for single attributes (like specifying the `lang` on something) whereas blocks are used wherever you pass HTML (like adding content after the opening body tag using `bodyStart`)

I don't think we've got it 100% consistent though unfortunately – `pageTitle` probably should be a variable rather than a block.

Reply:
Thanks for clarifying this! I think this helps because we can explain variables and blocks in Frontend docs and direct people there. I've had a rejig - let me know what you think.